code/_globalvars/lists/objects.dm

```diff
@@ -20,6 +20,7 @@ GLOBAL_LIST(chemical_reactions_list) //list of all /datum/chemical_reaction datu
 GLOBAL_LIST(chemical_reactions_list_reactant_index) //list of all /datum/chemical_reaction datums. Used during chemical reactions. Indexed by REACTANT types
 GLOBAL_LIST(chemical_reactions_list_product_index) //list of all /datum/chemical_reaction datums. Used for the reaction lookup UI. Indexed by PRODUCT type
 GLOBAL_LIST_INIT(chemical_reagents_list, init_chemical_reagent_list()) //list of all /datum/reagent datums indexed by reagent id. Used by chemistry stuff
+GLOBAL_LIST_INIT(chemical_name_list, init_chemical_name_list()) //names of reagents used by plumbing UI.
```

Comment:
```suggestion
/// names of reagents used by plumbing UI.
GLOBAL_LIST_INIT(chemical_name_list, init_chemical_name_list())
```

`///` == auto-doc comment. Yes I know the others are wrong

Reply:
Thanks